config/locales/models/home/tr.yml

```diff
@@ -0,0 +1,10 @@
+tr:
+  home:
+    components:
+      membership_notifications:
+        upcoming_events: Yaklaşan akademik takvim olayları
+        profile_completion_rate: Profil tamamlama oranı
+        address_information: Adres bilgileriniz
+        identity_information: Kimlik bilgileriniz
+        seems_missing: eksik görünüyor.
+        last_password_change: Son şifre değişikliği
```

Comment:
şifre vs parola.  buna da karar verip her yerde tutarlı kullanalım.  parola olsun diye ısrarcı değilim, şifre artık galat-ı meşhur oldu (ve malumunuz "galat-ı meşhur lugat-ı fasih'ten evladır").  çıkan kararı tutarlılık adına not etmek için yazıyorum.